packages/frontend/src/components/UserSettings/SlackSettingsPanel/SlackSettingsPanel.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { Button } from '@blueprintjs/core';
+import styled from 'styled-components';
+
+export const AppearancePanelWrapper = styled.div`
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Title = styled.h3`
+    margin: 0 0 10px;
+`;
+export const ColorPalette = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 40px;
+`;
+
+export const ColorSquare = styled.div`
+    height: 30px;
+    width: 30px;
+    padding: 0.286em;
```

Comment:
I find it really hard to maintain having em's in the codebase. any benefit of using it?

Reply:
Do you mean `em`s ? These tags are not even used, it was a copypaste leftover. I'll remove them.